android/build.gradle

```diff
@@ -20,7 +20,7 @@ android {
 
 dependencies {
     // Google's GCM.
-    compile 'com.google.android.gms:play-services-gcm:10.0.1'
+    compile "com.google.firebase:firebase-messaging:10.2.6"
```

Comment:
which classpath are you using to match this with 10.2.6?

`classpath "com.google.gms:google-services:3.0.0"` ? its crashing when using 3.1.0 (unsure why yet)

Reply:
Ah saw your note. Mine was crashing unless I included as follows: 

```
    compile 'com.google.android.gms:play-services-base:11.0.4'
    compile(project(':react-native-notifications')) {
        exclude group: 'com.google.firebase', module: 'firebase-messaging'
    }
    compile 'com.google.firebase:firebase-messaging:11.0.4'
```